app/assets/javascripts/smart-answers.js

```diff
@@ -1,7 +1,13 @@
+//= require start-button-ab-test-july-2016
+
 function browserSupportsHtml5HistoryApi() {
   return !! (history && history.replaceState && history.pushState);
 }
 
+function isExpectedPath(slug) { // Used mostly during A/B testing
```

Comment:
Your suggestion is for defining `SmartAnswer.isStartPage(‘overseas-passports’)` this way is also possible.